bammetrics.wdl

```diff
@@ -2,7 +2,9 @@ version 1.0
 
 import "tasks/picard.wdl" as picard
 import "tasks/samtools.wdl" as samtools
+import "tasks/biopet/bamstats.wdl" as bamstats
 import "tasks/common.wdl" as common
+import "tasks/biopet/bamstats.wdl" as bamstats
```

Comment:
```suggestion
```
I think importing it once is enough :stuck_out_tongue: 

Imports should also be in alphabetical order:
```wdl
import "tasks/biopet/bamstats.wdl" as bamstats
import "tasks/common.wdl" as common
import "tasks/picard.wdl" as picard
import "tasks/samtools.wdl" as samtools
```

Reply:
:+1: